test/odr_test.py

```diff
@@ -0,0 +1,48 @@
+import os
+import sys
+import re
+
+this_path = os.path.dirname(__file__)
+
+all_headers = set()
+include_path = this_path + "/../include/"
```

Comment:
```suggestion
include_path = os.path.join(this_path, "..", "include")
```

Reply:
(These are suggestions to help debug/fix - I have not tested locally and do not have a Windows machine)

Reply:
Good point, will do. I just applied that fix to iminuit's setup.py as well the other day.